boards/mec172xlj_mtl_template.conf

```diff
@@ -0,0 +1,35 @@
+# Zephyr-based EC FW
+
+# EC FW features configuration
+# ----------------------------
+CONFIG_ATX_SUPPORT=y
+
+CONFIG_PWRMGMT_DEEPSX=y
+
+# Enable PS2 device management but only keyboard due SoC port limitation
+CONFIG_PS2=y
+CONFIG_PS2_KEYBOARD=y
+CONFIG_PS2_MOUSE=n
+
+#Use UART0 for the eSPI communication
+#-------------------------------------
+CONFIG_ESPI_PERIPHERAL_UART_SOC_MAPPING=0
+
+# Disable PECI access in CS for desktop platforms.
+CONFIG_PECI_ACCESS_DISABLE_IN_CS=y
+
+# Support deprecated SMChost commands for backward compatibility
+CONFIG_DEPRECATED_SMCHOST_CMD=y
+
+# Enable gigadevice sequence
+CONFIG_SAF_ENABLE_XIP=y
+
+# Zephyr kernel/driver configuration required by EC FW
+# ----------------------------------------------------
+CONFIG_ESPI_PERIPHERAL_8042_KBC=y
+
+# Workaround to avoid overlap in SPI layout
+CONFIG_FLASH_SIZE=224
```

Comment:
what is this workaround ? is it due to SPI layout changes ?

Reply:
Yes, for compatibility with MTL-S IFWI layout